scripts/data-retention-remove-supplier-declarations.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Our data retention policy is that users have accounts deactivated and personal data stripped after 3 years without
+a login.
```

Comment:
A minor tweak needed to this, something like: `Our data retention policy is that supplier declarations are removed  3 years after the framework's expiry date`. We can extend this when we do the failed applications bit.